chrome-launcher/chrome-launcher.ts

```diff
@@ -94,14 +96,15 @@ export class Launcher {
     this.chromeFlags = defaults(this.opts.chromeFlags, []);
     this.requestedPort = defaults(this.opts.port, 0);
     this.chromePath = this.opts.chromePath;
+    this.enableExtensions = defaults(this.opts.enableExtensions, false);
   }
 
   private get flags() {
     const flags = DEFAULT_FLAGS.concat([
       `--remote-debugging-port=${this.port}`,
       // Place Chrome profile in a custom location we'll rm -rf later
       `--user-data-dir=${this.userDataDir}`
-    ]);
+    ]).filter(flag => this.enableExtensions ? flag !== '--disable-extensions' : true);
```

Comment:
@brendankenny Using a separate conditional statement it leads `const flags` to change into `let flags`. is that ok?  Or we can use splice instead of filter. 

```
let flags = ...
if (this.enableExtensions) {
   flags = flags.filter(flag => flag !== '--disable-extensions');
}
```